server/apps/entry/serializers.py

```diff
@@ -110,6 +112,12 @@ def validate_figures(self, figures):
             raise serializers.ValidationError('Duplicate keys found. ')
         return figures
 
+    def validate_reviewers(self, revs):
+        for reviewer in revs:
+            if not User.objects.filter(id=reviewer).exists():
+                raise serializers.ValidationError('Reviewer does not exist.')
+        return revs
```

Comment:
We could validate this using one query only, something like this.
```suggestion
        if User.objects.filter(id__in=revs).count() != len(revs):
            raise serializers.ValidationError('Reviewer does not exist.')
        return revs
```